main.py

```diff
@@ -47,10 +47,19 @@ def parse_args():
     sim_options = parser.add_argument_group(
         "simulation", "Options for data simulation")
     sim_options.add_argument("--simulate", "-s", action='store_true')
-    sim_options.add_argument("--data", '-d', default='sinus')
+    sim_options.add_argument(
+        "--data", '-d', default='sinus',
+        help="A path to a CSV file containing the ")
     sim_options.add_argument(
         "--error", "-e", type=float,
         help="The probability of error in each driver", default=0)
+    sim_options.add_argument(
+        "--sample-rate", "-r", help="The sample rate of the simulation data",
+        type=float, default=22)
+    parser.add_argument(
+        "--fps", "-f",
+        help="Frames-per-second for the application to render",
+        type=int, default=30)
```

Comment:
I think the default FPS should be lower (25) since on medical applications this is enough.
How about an option for the sample rate to be unlimited? We can achieve 60 SPS on the real module.
The white room guys wants at least 25 SPS i suggest changing the default to 30
```suggestion
    sim_options.add_argument(
        "--sample-rate", "-r", help="The sample rate of the simulation data",
        type=float, default=30)
    parser.add_argument(
        "--fps", "-f",
        help="Frames-per-second for the application to render",
        type=int, default=25)
```